Doc/c-api/init.rst

```diff
@@ -1112,10 +1112,19 @@ Sub-interpreter support
 
 While in most uses, you will only embed a single Python interpreter, there
 are cases where you need to create several independent interpreters in the
-same process and perhaps even in the same thread.  Sub-interpreters allow
-you to do that.  You can switch between sub-interpreters using the
-:c:func:`PyThreadState_Swap` function.  You can create and destroy them
-using the following functions:
+same process and perhaps even in the same thread. Sub-interpreters allow
+you to do that.
+
+The "main" interpreter is the first one created when the runtime initializes.
+The :c:func:`PyInterpreterState_Main` funtion returns a pointer to this main
+interpreter's state. It is usually the only Python interpreter in a process.
+The main interpreter also has other responsibilities like signal and public
+calls handling in the main thread, is responsible for execution during runtime
```

Comment:
You can drop "in the main thread".  It doesn't add much helpful info in this context.